src/vstart.sh

```diff
@@ -220,8 +220,8 @@ options:
 	-g --gssapi enable Kerberos/GSSApi authentication
 	-G disable Kerberos/GSSApi authentication
 	--hitset <pool> <hit_set_type>: enable hitset tracking
-	-e : create an erasure pool\
-	-o config		 add extra config parameters to all sections
+	-e : create an erasure pool
```

Comment:
Colon use in this list of options is inconsistent. There seem to be three different standards in play: (1) a comma separating the option from its explanation or gloss (206: "-d, --debug"), (2) nothing at all separating the option from its explanation or gloss (217: "-k keep old configuration files (default)"), and (3) a colon separating the option from its explanation or gloss (234: "--cache <pool>: enable cache tiering on pool").

I don't mean to minimize the authority of my station or diminish whatever standing documentation as an institution has in the open-source-software world, but I'll have to check with Josh and Neha before I provide approval of this since it's outside the ceph/doc/ directory.

Reply:
Also, Stapler seems to be causing the make check to fail with a 404.

Reply:
@jdurgin @neha-ojha: What's going on with this PR that's causing Stapler to participate in the failure of both the ceph API tests and the make check?

Reply:
Kinda weird. There is really no uniform standard, we could make it better.

Reply:
@Svelar: The lack of a standard for the format of flags and options isn't going to stop me from approving and merging this, when the required checks have passed. I'm mostly making a note to myself to return to this matter in a later commit and make this list consistent.

Reply:
jenkins retest this please

Reply:
> 

Of course, thanks a lot.

Reply:
After retest, all checks have passed.

Reply:
Any update?